### PR TITLE
[4.0] Restore com_admin profile view

### PR DESCRIPTION
**Pull request by SharkyKZ**
Pull Request for Issue https://github.com/joomla/joomla-cms/issues/25440.

### Summary of Changes

Restores profile view in `com_admin`.

### Testing Instructions

Test that editing profile using `Edit Account` link in `mod_user` dropdown works correctly. Especially functionality added by plugins (Token, TFA, action logs).

### Documentation Changes Required

If the removal was documented somewhere, it should be reverted.

### CODE REVIEW

File: administrator/components/com_admin/src/Controller/ProfileController.php [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+		$result = parent::save();
+
+		if ($this->getTask() !== 'apply')
+		{
+			$return = base64_decode($this->input->get('return', '', 'BASE64'));
+
+			if ($return !== '' && Uri::isInternal($return))
+			{
+			// Redirect to return URL.
+			$this->setRedirect(Route::_($return, false));
```
Comment (Quy) [inline administrator/components/com_admin/src/Controller/ProfileController.php]:
```suggestion
				// Redirect to return URL.
				$this->setRedirect(Route::_($return, false));
```

**Comment by dgrammatiko** [comment]
> Restores profile view in com_admin

May I ask why?

**Comment by Quy** [comment]
See https://github.com/joomla/joomla-cms/issues/25440

**Comment by brianteeman** [comment]
The problem we had before with com_admin was that not all user options would be loaded. With this PR for **example** then the accessibility tab isn't displayed (

**Comment by SharkyKZ** [comment]
>With this PR for example then the accessibility tab isn't displayed (

Because accessibility settings are hardcoded in `com_users` form. This is solved with https://github.com/joomla/joomla-cms/pull/29226. The alternatives are to use `com_users` form in `com_admin` or just copy/paste the settings into `com_admin` form.

**Comment by SharkyKZ** [comment]
@Quy should be fixed now.

**Comment by Quy** [comment]
Add password strength meter?

**Comment by brianteeman** [comment]
I cant help but think this is the wrong solution to the problem. Having to maintain the same code in multiple places....

**Comment by SharkyKZ** [comment]
@brianteeman There are ways of reducing code duplication here. We could use the form from com_users and TFA rendering could be moved to JLayout.

Besides, this contains some logic which is not present in com_users, like username compliance check.

**Comment by SharkyKZ** [comment]
@Quy Added.

**Comment by Quy** [comment]
Please fix conflicts.

**Comment by SharkyKZ** [comment]
Conflicts fixed.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 766657a1a5bda55d98fe111e3096a5f40f22bd2e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29251">issues.joomla.org/tracker/joomla-cms/29251</a>.</sub>

**Comment by infograf768** [comment]
Conflicts with admin.ini after my alpha order pr.
Also, what do we do with `JHELP_ADMIN_USER_PROFILE_EDIT="Site_My_Profile"` that is used in 3.x ?
See https://github.com/joomla/joomla-cms/issues/29687

**Comment by SharkyKZ** [comment]
Conflicts fixed. `JHELP_ADMIN_USER_PROFILE_EDIT` can be removed for now. I'll just have to re-add it in this PR.

**Comment by infograf768** [comment]
> JHELP_ADMIN_USER_PROFILE_EDIT can be removed for now. 

Not necessary as https://github.com/joomla/joomla-cms/issues/29687 is not a PR but an issue.

**Comment by infograf768** [comment]
Are we supposed to get an Action in Home Dashboard Latest Actions when modifying profile?

If I modify the user account, I do get a log.

**Comment by SharkyKZ** [comment]
@infograf768 I guess we could add this. Currently doesn't work in staging either.

**Comment by infograf768** [comment]
@SharkyKZ 
In fact I thought there was something missing in your pr as it also modifies actionlogs.
But yes, it makes sense to add it (never tested in 3.x).

**Comment by jwaisner** [comment]
@SharkyKZ  is the action logs for the account modification going to be apart of this PR or will this be separate? Everything else tests out good, just the action logs that do not record the edits.

**Comment by SharkyKZ** [comment]
Maybe separate. This could be fixed in staging too.

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 766657a1a5bda55d98fe111e3096a5f40f22bd2e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29251">issues.joomla.org/tracker/joomla-cms/29251</a>.</sub>

**Comment by jwaisner** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29251">issues.joomla.org/tracker/joomla-cms/29251</a>.</sub>

**Comment by brianteeman** [comment]
i still think is the wrong solution

**Comment by SharkyKZ** [comment]
What would be the right solution?

**Comment by infograf768** [comment]
Tks.

**Pull request merged**